api/discovery/discovery.proto

```diff
@@ -84,7 +84,7 @@ message Resource {
   // Id contains a unique ID for each resource. This is specific for the cloud
   // provider this resource was gathered for and can for example be a resource
   // URL.
-  string id = 1 [ (validate.rules).string.uuid = true ];
+  string id = 1 [ (validate.rules).string.min_len = 1 ];
```

Comment:
Interesting, how did it work before ^^ Resource IDs in Azure (starting with "/subscriptions") definitely weren't UUIDs

Reply:
It probably worked because we never added "Resource" objects from the outside, only the ones inside an evidence and those are different and not validated.

Reply:
Right :D